### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by dabeeeenster**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/blank-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Blank issue template
```
Comment (matthewelwell) [inline .github/ISSUE_TEMPLATE/blank-issue-template.md:2]: I wonder if people will just use this to bypass the templates? But I guess we would want to use this template ourselves?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**How are you running Flagsmith?**
+- [Self Hosted with Docker]
```
Comment (matthewelwell) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should these be checkboxes?

File: .github/ISSUE_TEMPLATE/blank-issue-template.md (hunk 1 above)
Comment (dabeeeenster) [reply .github/ISSUE_TEMPLATE/blank-issue-template.md:2]: Yep that was my thinking - can see how it goes

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (dabeeeenster) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yep fixed

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**